packages/forms/style.ts

```diff
@@ -0,0 +1,101 @@
+import { css } from "emotion";
+import { coreColors } from "../shared/styles/color";
+
+const {
+  green,
+  greenLighten5,
+  greyDark,
+  greyDarkLighten3,
+  greyLight,
+  greyLightLighten3,
+  purple,
+  purpleLighten5,
+  red,
+  redLighten5,
+  white
+} = coreColors();
+
+export const errorColor = red;
+
+export const inputAppearances = {
+  standard: css`
+    background-color: ${white};
+    border-color: ${greyLight};
+    svg {
+      fill: ${greyDark};
+    }
+    &:focus {
+      background-color: ${purpleLighten5};
+      border-color: ${purple};
+      svg {
+        fill: ${purple};
+      }
+    }
+  `,
+  disabled: css`
+    background-color: ${greyLightLighten3};
+    border-color: ${greyLightLighten3};
+    svg {
+      fill: ${greyDarkLighten3};
```

Comment:
@mperrotti or @lilysamimi this was a total guess on my part so that icons are still displayed when the input is disabled, but are lighter then normal. Is that acceptable or desired?

Reply:
We should try and normalize the text and icon colors. Set `color: ${whateverColor}`, use the [::placeholder selector](https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder), use the same color for all three: icon, placeholder text, and value text

Reply:
Shouldn't placeholder text be lighter then value text? 

<img width="454" alt="image" src="https://user-images.githubusercontent.com/1972968/49475669-6dfe1000-f7dd-11e8-88a5-262e6b0738d2.png">